### DISCUSSION TITLE
Welcome to Vendure Discussions!

### DISCUSSION THREAD

**Discussion by michaelbromley** [vendurehq/vendure#468] [General]
## 👋 Welcome!

Hi!

We're trying out the new GitHub Discussions beta. We already have a [Slack workgroup](https://join.slack.com/t/vendure-ecommerce/shared_invite/enQtNzA1NTcyMDY3NTg0LTMzZGQzNDczOWJiMTU2YjAyNWJlMzdmZGE3ZDY5Y2RjMGYxZWNlYTI4NmU4Y2Q1MDNlYzE4MzQ5ODcyYTdmMGU) which is great for conversations and quick questions. But as a repository of collective knowledge about Vendure, it is no good since it is not publicly discoverable, and content disappears after a while.

So here's the place to post questions or discussions about Vendure that may prove useful to others. 

Here's the place to discuss things like:

* Plugin development
* Production deployment
* Project architecture
* Design of the framework
* Anything else that might be useful in a publicly-discoverable knowledge base about Vendure.

Let's see how it goes!

**Comment by agustif** [comment]
Hello, this is awesome news!
